README.md

```diff
@@ -1 +1,55 @@
-# to-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do-list
+
+> This is my first project using webpack in the Microverse. The objective was to build a to-do-list by sticking with some design criteria, some ES6 modules and syntax. For that I created to-do-list repo and cloned into my local machine. For best practices I have implemented Linter,Github Flow and professional documentation as much as I can. 
+
+![screenshot](./images/screenshot.png)
+
+## Built With
+
+- HTML, CSS, js
+- Linters, Markdown
+- GitHub Flow 
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1) Enter `git@github.com:basnetlaxmi/to-do-list.git` into terminal
+
+2) Open in browser to view site
+
```

Comment:
## Optional
- [ ] Please add each and every step to get the app locally up and running you can simply add after cloning:
- `npm i`
- `npm run build`
- `npm run start`